### PR TITLE
Check translation progress during final release

### PR DESCRIPTION
**Pull request by loremattei**
This PR adds a step to the `finalize_release` lane to check the current state of the translation.  
If some of the languages which are expected to be fully translated at release time are below 100%, the lane outputs the current state of the translation and asks confirmation to the user before continuing. 

### Example: 

```
[15:53:43]: ----------------------------------------
[15:53:43]: --- Step: check_translation_progress ---
[15:53:43]: ----------------------------------------
[15:53:43]: Checking translations status...
[15:53:44]: > Getting translation status for ar
[15:53:44]: Language ar is 99% translated.
[15:53:44]: > Getting translation status for de
[15:53:44]: Language de is 97% translated.
[15:53:44]: > Getting translation status for es
[15:53:44]: Language es is 100% translated.
[15:53:44]: > Getting translation status for fr
[15:53:44]: Language fr is 96% translated.
[15:53:44]: > Getting translation status for he
[15:53:44]: Language he is 97% translated.
[15:53:44]: > Getting translation status for id
[15:53:44]: Language id is 96% translated.
[15:53:44]: > Getting translation status for it
[15:53:44]: Language it is 97% translated.
[15:53:44]: > Getting translation status for ja
[15:53:44]: Language ja is 99% translated.
[15:53:44]: > Getting translation status for ko
[15:53:44]: Language ko is 97% translated.
[15:53:44]: > Getting translation status for nl
[15:53:44]: Language nl is 100% translated.
[15:53:44]: > Getting translation status for pt-br
[15:53:44]: Language pt-br is 98% translated.
[15:53:44]: > Getting translation status for ru
[15:53:44]: Language ru is 100% translated.
[15:53:44]: > Getting translation status for sv
[15:53:44]: Language sv is 100% translated.
[15:53:44]: > Getting translation status for tr
[15:53:44]: Language tr is 99% translated.
[15:53:44]: > Getting translation status for zh-cn
[15:53:44]: Language zh-cn is 100% translated.
[15:53:44]: > Getting translation status for zh-tw
[15:53:44]: Language zh-tw is 97% translated.
[15:53:44]: The translations for the following languages are below the 100% threshold:
 - ar is at 99%.
 - de is at 97%.
 - fr is at 96%.
 - he is at 97%.
 - id is at 96%.
 - it is at 97%.
 - ja is at 99%.
 - ko is at 97%.
 - pt-br is at 98%.
 - tr is at 99%.
 - zh-tw is at 97%.
Do you want to continue? (y/n)
``` 

## Regression Notes
1. Potential unintended areas of impact
-

2. What I did to test those areas of impact (or what existing automated tests I relied on)
-

3. What automated tests I added (or what prevented me from doing so)
-

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by oguzkocer — APPROVED** [review]
Looks good to me - I'll leave it to @jkmassel to merge as the release manager.

**Comment by mokagio** [comment]
I merged `develop` into this to address a conflict on `Gemfile.lock` and enabled auto-merge.

This is a PR I _definitely_ want into `develop` before the 17.6 code freeze 😄

**Pull request merged**
